### ISSUE TITLE
Composing Containers

### ISSUE THREAD

**Issue by arunoda** [facebook/react#5770]
When building some apps, I found out containers plays a major roll in our apps. Of course Relay and Redux make it easy to create them. But for me, there are some other data sources (and caches) which manages the state of our apps.

So, having a clean and clear way to compose Containers is important for me. That's why I created [react-komposer](https://github.com/kadirahq/react-komposer).

See: [Let’s Compose Some React Containers](https://voice.kadira.io/let-s-compose-some-react-containers-3b91b6d9b7c8#.qpxo5k2hk)

Basically, with that you could compose from a new kind data source, including promises, Rx.js observables & even Meteor's tracker based data sources.

Check some demos: <https://github.com/kadirahq/react-komposer#using-with-xxx>

Looking forward for feedback on this.

**Comment by zpao** [comment]
Please keep github for bugs in React. There are other places for posting your related projects, like https://discuss.reactjs.org/

**Comment by arunoda** [comment]
Oh! Sorry about that.
